### DISCUSSION TITLE
Hide Navigation from the public

### DISCUSSION THREAD

**Discussion by lemonwater** [requarks/wiki#7411] [Help / Questions]
Can we hide the Navigation (The Sidebar on the left) from the public and show only for login users?  I can use Static Navigation to do similar effect except when public user access the wiki, an empty side bar was shown.  There is a way to disable Navigation.  So will it be possible to enable and disable Navigation accordingly?

**Comment by craigreyenga** [comment]
It seems reasonable that if the side bar is completely empty, it should disable itself automatically.
